docs/concepts/portal/installation-portal.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Ondat Portal Installation Guide"
+linkTitle: "Ondat Portal Installation Guide"
+---
+
+
+# Setting Up a Local Cluster 
+
+The following guide will take you through the process of setting up a cluster on your machine. If you are using an Apple M1 machine you need to run kind inside an amd64 VM hosted on a Cloud Service of your choice. Local emulation will not work on a machine using Apple M1. 
+
+## Prerequisites
+
+* You have set up your cluster using the following config file <cloud_init.yml>
```

Comment:
Is the file or its contents included somewhere linkable in the documentation?

Reply:
No, actually no idea where it is. If you have it and you can help with that - great. If not have to do some investigation cause I do not have the config copied locally. Thanks